src/argilla/training/setfit.py

```diff
@@ -13,13 +13,16 @@
 #  limitations under the License.
 
 import json
+import logging
 from typing import List, Union
 
 from argilla.training.transformers import ArgillaTransformersTrainer
 from argilla.training.utils import filter_allowed_args, get_default_args
 
 
 class ArgillaSetFitTrainer(ArgillaTransformersTrainer):
+    _logger = logging.getLogger("argilla.training.ArgillaSetFitTrainer")
```

Comment:
Is this equivalent to `getLogger(__init__)`? I tend to see that pattern, but this is likely just as fine.